de.metas.util/src/main/java/de/metas/util/rest/ExternalHeaderAndLineId.java

```diff
@@ -0,0 +1,23 @@
+package de.metas.util.rest;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class ExternalHeaderAndLineId {
+
+	private List<ExternalId> externalLineIds;
+	private String externalHeaderId;
```

Comment:
same as with the cossreponsing JSON-class. i don't see why whe have `ExternalId` for the lineIds (which i prefer), but `String``for the headerIds

Reply:
I have encountered some problems when trying to replace String with ExternalId which involved longer effort to fix it so I have decided to leave it as it is. In JsonOLCandCreateRequest we use String fro both externalId and externalHeaderId, if we decide to change this, maybe that's also a place to change it as well.